lmdeploy/pytorch/engine/model_agent.py

```diff
@@ -517,7 +517,8 @@ def _check_context_alive(mp_context: mp.ProcessContext):
     for idx, exitcode in log_procs:
         logger.error(f'TP process {idx} failed with exitcode {exitcode}.')
     # TODO: not safe exit.
```

Comment:
still `not safe exit`?

Reply:
It's safer to me. @grimoire can we remove this comment?

Reply:
`os._exit` is not safe.